src/client/containers/authentication/authentication.js

```diff
@@ -15,24 +15,20 @@ class Authentication extends Component {
     };
 
     onInputChange = ({ target: { name, value } }) => {
-      this.setState(({ userData }) => {
-        userData[name] = value;
-        return userData;
-      });
+      const userData = Object.assign({}, this.state.userData);
```

Comment:
Instead of object.assing, you could use spread operator here const userData = {...this.state.userData }
Or even use it in set state func

Reply:
done